src/Sylius/Behat/Context/Ui/Shop/LocaleContext.php

```diff
@@ -40,6 +47,14 @@ public function iSwitchTheLocaleToTheLocale(string $locale): void
         $this->homePage->switchLocale($locale);
     }
 
+    /**
+     * @When I show homepage with the locale :locale
```

Comment:
We should change this implementation to:
```suggestion
     * @When I show homepage with the locale :localeCode
```
or 
```suggestion
     * @When I show homepage with the locale :language
```

So `\Sylius\Behat\Context\Transform\LocaleContext` will convert value for us.